README.md

```diff
@@ -100,5 +100,25 @@ The most important interactions with the contracts are creation of identities an
 #### Send a meta-tx
 ![meta-tx](./docs/diagrams/send-tx.seq.png)
 
+## Deploying contracts to a private network
+Add a new entry in `./truffle.js` like so:
+```js
+yourNetwork: {
+  get provider() {
+    return getProvider("http://my-private.network/")
+  },
+  network_id: "*"
+},
+```
+If the gas price on the network is very high you might need to add the `gasPrice` parameter.
+
+Create a file called `./seed`. This file should contain a mnemonic seed phrase. Make sure that the first address of this seed has ether on your private network. Then run:
```

Comment:
@oed what does "first address of this seed" mean?
The master key, `m/` or the bip44 eth standard derivation `m/44'/60'/0'/0`?

Reply:
Good point! Yes it should be `m/44'/60'/0'/0` let me update the readme!